### ISSUE TITLE
sift not in debian-repository

### ISSUE THREAD

**Issue by seitza** [Rostlab/SNAP2#3]
I tried to install the snap2 package with the routine posted on https://rostlab.org/owiki/index.php/Debian_repository .

However, it stopped working, while checking for the dependencies. Namely, sift (>4.0.3) was missing.
I checked, if it should be available in the repository according to https://rostlab.org/owiki/index.php/Packages and it should. Furthermore, I tried to install it manually to satisfy the dependency. There are two external links given in the package list: http://sift.jcvi.org and [Debian Med](http://blends.debian.org/med/tasks/bio).

Therefore I downloaded the tar.gz from the first http://sift.jcvi.org/www/www/sift4.0.3b.tar.gz. However this package seems only to wrap around the real sift, as the only perl script, which could be called the main executable tries to access another script called sift3.0. The README only contains information about genome wide analysis (throwing a great number of seuqences against the DB) and contains no trace about the original "smaller" sift (only one sequence against the DB).

The same accounts for the tarsal from http://blends.debian.org/med/tasks/bio .
Then I checked out why the debian repo fails, and obviously sift is still in the repo, but I cant install it ...

I'm done for tonight ...

**Comment by juanmirocks** [comment]
:+1: :) -- Max is informed

**Comment by hechtm** [comment]
I think sift is not available for lucid at the moment.
For all others sift is available in the non-free section of the rostlab debian repository. Simply do:
sudo bash -c 'echo -e "deb http://rostlab.org/debian/ stable main contrib non-free\ndeb-src http://rostlab.org/debian/ stable main contrib non-free" > /etc/apt/sources.list.d/rostlab.list'
as described in https://rostlab.org/owiki/index.php/Debian_repository

**Comment by seitza** [comment]
Hi @hechtm ,

you wrote, sift might not be available for lucid atm. Did you mean lucid or did you mean wheezy? For testing I use the debian distribution with version wheezy 7.something. I followed the instructions from rostlab.org about installing and running packages (https://rostlab.org/owiki/index.php/Debian_repository). I executed the following code:
- sudo apt-get install python-software-properties
- sudo apt-add-repository "deb http://rostlab.org/debian/ stable main contrib non-free"
- sudo apt-get update
- sudo apt-get install rostlab-debian-keyring
- sudo apt-get update
- sudo apt-get install snap2

All went well until i got

```
The following packages have unmet dependencies:
 snap2 : Depends: sift (>= 4.0.3b)
```

I checked if the package repository lists contain information about sift, and unfortunately they do not. So I think its not possible to install sift from rostlab.org. I tried to install it manually, as already stated in the original post, but failed. I tried two more times since then on new VMs but failed as well.  

Any ideas? In best case how to solve the problem via the rostlab repo? thx
